packages/legacy-client/src/client/actions/createActionProxy.ts

```diff
@@ -51,6 +51,19 @@ export function createActionProxy<
 
         return undefined;
       },
+      ownKeys(_target) {
+        return Object.keys(client.ontology.actions);
+      },
+      getOwnPropertyDescriptor(_target, p) {
```

Comment:
Might be worth just constructing an object or a prototype in the future given we have to do so much work anyway.

Reply:
On second thought, probably not. A proxy means we can potentially support calls that we don't know about at compile time.